File: uploaddemo/src/business/galary/java/com/jw/galary/img/loader/ImageLoader.java


package com.jw.galary.img.loader;

import android.app.Activity;
import android.widget.ImageView;
import java.io.Serializable;

public interface ImageLoader extends Serializable {
  void displayImage(Activity var1, String var2, ImageView var3, int var4, int var5);

  void displayImagePreview(Activity var1, String var2, ImageView var3, int var4, int var5);

  void clearMemoryCache();
}

File: library/src/main/java/com/jw/library/loader/ImageLoader.kt
package com.jw.library.loader

import android.content.Context
import android.widget.ImageView
import java.io.Serializable

interface ImageLoader : Serializable {
    fun displayImage(
        context: Context,
        path: String,
        imageView: ImageView
    )

    fun displayImageRotate(
        context: Context,
        path: String,
        imageView: ImageView,
        orientation: Int
    )

    fun clearMemoryCache()
}
